src/charactersheet/viewmodels/common/media_manager/index.html

```diff
@@ -0,0 +1,29 @@
+<div class="mt-2">
+  <flip-card params="{
+    tabId: tabId,
+    elementId: `${tabId}_${Math.random() * 100000 | 0}_media_list`,
```

Comment:
what's up with the random here?

Reply:
It's essential that every instance of the media manager have its own unique ID for the flip cards to work.

Reply:
ah, I see.